packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js

```diff
@@ -11,6 +11,14 @@ export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
     this.$stateParams = $stateParams;
     this.TucToast = TucToast;
     this.OvhApiTelephony = OvhApiTelephony;
+
+    this.currency = '€'; // No currency given by API (FR only product)
+
+    this.getRepayementsTotalPrice = (repayments) =>
+      repayments.reduce(
+        (total, repayment) => total + repayment.price * 1000,
+        0,
+      ) / 1000;
```

Comment:
why *1000 each variables and /1000 after ? It give the same result no ?

Reply:
in js variable are like float no ? (no type like unssigned or int).

Reply:
In javascript, decimal manipulation may have unwanted side effects. so this trick is to avoid these side effects by manipulating integers, and once the calculation is done, re format the number to the wanted decimal

Reply:
You're right, it's for decimal (3 in this case).

```javascript
0.1 + 0.2 // 0.30000000000000004

(0.1*10 + 0.2*10)/10 // 0.3
```